### PR TITLE
fix list not showing multiple releases with same name in different namespaces

### LINKED ISSUE #6658: Helm3: list all namespaces not showing multiple releases with same name in different namespaces
I created two releases with same name in two different namespaces, and then tried to list releases in all namespaces, but only one of them showed up instead of two showing up

Output of `helm version`:
```
version.BuildInfo{Version:"v3.0.0-beta.4", GitCommit:"7ffc879f137bd3a69eea53349b01f05e3d1d2385", GitTreeState:"dirty", GoVersion:"go1.13.1"}
```

### PR DESCRIPTION
**Pull request by karuppiah7890**
**What this PR does / why we need it**:
Fixes #6658 
Twin of #6662 which got closed due to branch renaming

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [X] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by karuppiah7890** [comment]
@thomastaylor312 I've added a simple test for this fix. How does it look?

**Review by thomastaylor312 — APPROVED** [review]
One minor nit here that won't block merge, but I'll keep this open for a little bit longer in case you'd like to address it

File: pkg/action/list_test.go [hunk 1]
```diff
@@ -159,14 +159,16 @@ func TestList_LimitOffsetOutOfBounds(t *testing.T) {
...
 func TestList_StateMask(t *testing.T) {
 	is := assert.New(t)
 	lister := newListFixture(t)
 	makeMeSomeReleases(lister.cfg.Releases, t)
 	one, err := lister.cfg.Releases.Get("one", 1)
 	is.NoError(err)
 	one.SetStatus(release.StatusUninstalled, "uninstalled")
-	lister.cfg.Releases.Update(one)
+	err = lister.cfg.Releases.Update(one)
+	assert.NoError(t, err)
```
[the remark this answers is not in the document]
Comment (karuppiah7890) [reply pkg/action/list_test.go]: Changed! :)

**Review by hickeyma — APPROVED** [review]
I tested this manually also and it works as expected. Nice fix, thanks @karuppiah7890. LGTM.

**Pull request merged**